test/data_constructors.jl

```diff
@@ -1,105 +1,311 @@
 @testset "data_constructors.jl" begin
     @testset "data_response" begin
-        # NamedTuples
-        f = @formula(y_int ~ x_float + x_cat)
-        y = TuringGLM.data_response(f, nt_str)
-        @test y == [2, 3, 4, 5]
-        y = TuringGLM.data_response(f, nt_cat)
-        @test y == [2, 3, 4, 5]
+        @testset "NamedTuples" begin
+            f = @formula y_int ~ 0 + x_float + x_cat
+            y = TuringGLM.data_response(f, nt_str)
+            @test y == [2, 3, 4, 5]
+            y = TuringGLM.data_response(f, nt_cat)
+            @test y == [2, 3, 4, 5]
+            f = @formula y_int ~ 1 + x_float + x_cat
+            y = TuringGLM.data_response(f, nt_str)
+            @test y == [2, 3, 4, 5]
+            y = TuringGLM.data_response(f, nt_cat)
+            @test y == [2, 3, 4, 5]
```

Comment:
Why so much code duplication with the `[2, 3, 4, 5]`? Same comment holds in increasing priority (increasing duplication) below

Reply:
I am trying to test NamedTuples/DataFrames using strings/categorical vectors. I want to make the tests sensible to any breaking change.

Reply:
Oh! Now I get it!